tools/migrate/migrate-l10n.py

```diff
@@ -38,6 +37,12 @@ def main(lang, reference_dir, localization_dir, blame, migrations, dry_run):
         # Keep track of how many changesets we're committing.
         index = 0
 
+        # Annotate legacy localization files used as sources by this migration
+        # to preserve attribution of translations.
+        files = ctx.localization_resources.keys()
+        blame = Blame(localization_dir).main(files)
```

Comment:
I think we should pass the hglib client to Blame directly. That way, we only have one hg process open instead of ... n + 1 minus whatever gets garbage-collected?
 
Either in this patch or in a follow-up, I think.